buildSrc/src/main/groovy/spring.jdbc.plus.maven-publish-conventions.gradle

```diff
@@ -1,29 +1,58 @@
 plugins {
     id "maven-publish"
     id "signing"
+    id 'org.jreleaser'
 }
 
 group = "com.navercorp.spring"
 version = "3.5.0"
 
+jreleaser {
+    release {
+        github {
+            skipRelease = true
+        }
+    }
+    gitRootSearch = true
+    signing {
+        active = 'ALWAYS'
+        armored = true
+        verify = true
+    }
+    deploy {
+        maven {
+            mavenCentral {
+                'release-deploy' {
+                    active = 'RELEASE'
+                    url = 'https://central.sonatype.com/api/v1/publisher'
+                    sign = true
+                    stagingRepository('build/staging-deploy')
+                }
+            }
+            // The following configuration is applied when the version matches the "*-SNAPSHOT" pattern.
+            nexus2 {
+                'snapshot-deploy' {
+                    active = 'SNAPSHOT'
+                    snapshotUrl = 'https://central.sonatype.com/repository/maven-snapshots/'
+                    applyMavenCentralRules = true
+                    snapshotSupported = true
+                    closeRepository = true
+                    releaseRepository = false
+                    stagingRepository('build/staging-deploy')
+                }
+            }
```

Comment:
jreleaser automatically choose the deployment destination based on the version name.

You can verify this behavior by running `./gradlew jreleaserConfig` with a version that includes a `-SNAPSHOT` suffix.